tests/unit/misc/test_sql.py

```diff
@@ -120,23 +120,26 @@ def test_text(self, klass):
         assert err.text() == "db text"
 
 
-def test_init():
-    sql.SqlTable('Foo', ['name', 'val', 'lucky'])
+def test_init_table(data_tmpdir):
+    db = sql.Database(str(data_tmpdir / 'test_init_table.db'))
```

Comment:
I suppose the `str` here can be dropped entirely, or perhaps replaced by `pathlib.Path` now?

Note that `data_tmpdir` is a [py.path.local](https://py.readthedocs.io/en/latest/path.html) object for historical reasons - but the API is compatible to pathlib for some simple cases, so maybe we can get away with not converting it at all.

Reply:
Ah, nevermind, I only now saw your comment on leaving `sql.Database.__init__` as-is.